option.go

```diff
@@ -282,6 +294,25 @@ func (option *Option) Set(value *string) error {
 	return convert("", option.value, option.tag)
 }
 
+func (option *Option) SetTerminatedOption(value []string) error {
```

Comment:
Does this need to be exported (capitalized)? If so, it should have a docstring like `Set`. The name also seems a bit misleading as it appends rather than sets.

Reply:
I think it does. Because, for terminated options, `Set(*string)` won't work. 
I have added a docstring and updated the function as well.